dev-php/pecl-parallel/Manifest

```diff
@@ -0,0 +1 @@
+DIST parallel-1.1.4.tgz 58783 BLAKE2B 942c767ff4838c13e70829d6ac6599cee3185afaf71e9dc430064835102d31a3e50b99694e1632be652c6a019934a13ba0f5139041bba39d486a20615478b910 SHA512 6693f01b0d43de5153aa0953f90580717e519cfbe8ca5b0ba6abd9cc681e0efe9d7c6fcc601da98f74b570b5810dffdbf5100bedc8ab86c1f4bc61d48955ae77
```

Comment:
Is there an easy way to rename this as `${PN}`? I'd really prefer that. Rest looks good, as always...

Reply:
I can see why ... doesn't look like there is a simple way no, this is as per the php-ext-pecl-r3 eclass.

So fortunately it doesn't look like there are any *probable* collisions in the tree (dev-ruby/parallel uses .tar.gz not .tgz and sys-process/parallel uses .tar.bz2).  Confusing yes ...

Perhaps we should consider updating the eclass to rename on download?

51 if [[ -z "${PHP_EXT_PECL_FILENAME}" ]] ; then
52     SRC_URI="https://pecl.php.net/get/${PHP_EXT_PECL_PKG_V}.tgz"
53 else
54     SRC_URI="https://pecl.php.net/get/${PHP_EXT_PECL_FILENAME}"
55 fi

-> ${P}.tgz and -> ${P}.${PHP_EXT_PECL_FILENAME##*.} respectively?  Although, the latter would need more care for double extensions such as .tar.${comp}

Alternatively a PHP_EXT_PECL_DOWNLOADED_FILENAME which if not set we default to ${P}.tgz if PHP_EXT_PECL_FILENAME is not set, or ${P}.${detected_ext} ... where detected_ext is tar.${PHP_EXT_PECL_FILENAME##*.} or simply ${PHP_EXT_PECL_FILENAME##*.} ... but this gets nasty no matter how you look at it.